basicsettings.ml

```diff
@@ -24,36 +24,32 @@ let concurrent_server = Settings.add_bool ("concurrent_server", true, `System)
 (** Set this to [true] to print types when printing results. *)
 let printing_types = Settings.add_bool ("printing_types", true, `User)
 
-(* Looks for a given file, either in the current directory or in the Links opam path *)
```

Comment:
I'm not sure of the reason for this change, is it undoing reorganization that was done concurrently with your branch?